lib/rubocop/cop/rails/time_zone.rb

```diff
@@ -14,16 +14,34 @@ module Rails
       # When EnforcedStyle is 'flexible' then it's also allowed
       # to use Time.in_time_zone.
       #
-      # @example
-      #   # always offense
+      # @example EnforcedStyle: strict
+      #   # The value `strict` means that `Time` should be used with `zone`.
+      #
+      #   # bad
+      #   Time.now
+      #   Time.parse('2015-03-02 19:05:37')
+      #
+      #   # bad
+      #   Time.current
+      #   DateTime.strptime(str, "%Y-%m-%d %H:%M %Z").in_time_zone
```

Comment:
It seems to be a missing fix for Issue #1773, but it seems there was a mistake in solution itself of that issue. I will open another PR on that issue.